app/Providers/MugShotServiceProvider.php

```diff
@@ -6,54 +6,61 @@
 
 use Illuminate\Support\ServiceProvider;
 use Spatie\Browsershot\Browsershot;
-use Spatie\Image\Manipulations;
 
-class MugShotServiceProvider  extends ServiceProvider
+class MugShotServiceProvider extends ServiceProvider
 {
     protected string $configFile = '/config/mugshot.php';
 
-    /**
-     * Register the service provider.
-     *
-     * @return void
-     */
     public function register()
     {
         $this->app->singleton(Browsershot::class, function ($app) {
             $config = $app->make('config')['mugshot'];
 
-            $instance = new Browsershot();
-            $instance
-                ->setNodeBinary($config['puppeteer']['node'])
-                ->setNpmBinary($config['puppeteer']['npm'])
-                ->setProxyServer($config['puppeteer']['proxyServer'])
-                ->setChromePath($config['puppeteer']['chrome'])
+            $instance = (new Browsershot())
+                ->timeout($config['timeout'])
                 ->userAgent($config['request']['useragent'])
-                ->ignoreHttpsErrors()
-            ;
+                ->ignoreHttpsErrors();
+
+            if (!$config['puppeteer']['sandbox']) {
+                $instance->noSandbox();
+            }
+            if (!is_null($config['puppeteer']['node'])) {
```

Comment:
You should use empty checks here. In the `.env.example` file you are using empty strings by default, but if you would use them in runtime, these empty strings would be used here.